boards/arm/96b_meerkat96/doc/index.rst

```diff
@@ -0,0 +1,330 @@
+| LED_R1        | GPIO1_IO04      | LED0                      |
++---------------+-----------------+---------------------------+
+| LED_R2        | GPIO1_IO05      | LED1                      |
++---------------+-----------------+---------------------------+
+| LED_R3        | GPIO1_IO06      | LED2                      |
++---------------+-----------------+---------------------------+
+| LED_R4        | GPIO1_IO07      | LED3                      |
++---------------+-----------------+---------------------------+
+
+System Clock
+============
+
+The M4 Core is configured to run at a 200 MHz clock speed.
+
+Serial Port
+===========
+
+The iMX7D SoC has seven UARTs. The number 2 is configured for the console and
+the remaining are not used/tested.
+
+Programming and Debugging
+*************************
+
+The 96Boards Meerakat96 board doesn't have QSPI flash for the M4 and it needs
+to be started by the A7 core. The A7 core is responsible to load the M4 binary
+application into the RAM, put the M4 in reset, set the M4 Program Counter and
+Stack Pointer, and get the M4 out of reset. The A7 can perform these steps at
+bootloader level or after the Linux system has booted.
+
+The M4 can use up to 5 different RAMs. These are the memory mapping for A7 and M4:
+
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| Region     | Cortex-A7             | Cortex-M4 (System Bus) | Cortex-M4 (Code Bus)  | Size                 |
++============+=======================+========================+=======================+======================+
+| DDR        | 0x80000000-0xFFFFFFFF | 0x80000000-0xDFFFFFFF  | 0x10000000-0x1FFEFFFF | 2048MB (less for M4) |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| OCRAM      | 0x00900000-0x0091FFFF | 0x20200000-0x2021FFFF  | 0x00900000-0x0091FFFF | 128KB                |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| TCMU       | 0x00800000-0x00807FFF | 0x20000000-0x20007FFF  |                       | 32KB                 |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| TCML       | 0x007F8000-0x007FFFFF |                        | 0x1FFF8000-0x1FFFFFFF | 32KB                 |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| OCRAM_S    | 0x00180000-0x00187FFF | 0x20180000-0x20187FFF  | 0x00000000-0x00007FFF | 32KB                 |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| QSPI Flash |                       |                        | 0x08000000-0x0BFFFFFF | 64MB                 |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+
+
+References
```

Comment:
Because the subsequent paragraphs are not part of this "References" section, I'd delete this heading and just write this as:
```
For more information about memory mapping see the
`i.MX 7 Dual Reference Manual`_  (section 2.1.2 and 2.1.3), and the
`Toradex Wiki`_.
```